azure-pipelines.yml

```diff
@@ -0,0 +1,113 @@
+variables:
+  vcpkgGitRef: 1be75a28bfce2084bb0b59ec92a6853f13d7be66 # Update this git ref to update vcpkg
+
+jobs:
+- job: MSVC_64
+  displayName: 'MSVC 64bit'
+  pool:
+    vmImage: 'windows-latest'
+  timeoutInMinutes: 250
+
+  steps:
+  - checkout: self
+    submodules: true
+    persistCredentials: true
+
+  - task: Cache@2
+    displayName: 'Cache VCPKG Artifacts'
+    inputs:
+      key: $(Build.SourcesDirectory)/msvc-x64-deps.txt | "$(vcpkgGitRef)" | "$(Agent.OS)"
+      path: '$(Build.BinariesDirectory)/vcpkg'
+  - task: run-vcpkg@0
+    displayName: 'Run VCPKG'
+    inputs:
+      vcpkgArguments: "@$(Build.SourcesDirectory)/msvc-x64-deps.txt"
+      vcpkgGitCommitId: $(vcpkgGitRef)
+      vcpkgGitURL: https://github.com/microsoft/vcpkg.git
+
+  - powershell: |
+      mkdir $(Build.BinariesDirectory)/Release
+      cd $(Build.BinariesDirectory)/Release
+      cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=MinSizeRel -DVCPKG_ROOT=$(Build.BinariesDirectory)\vcpkg -DVCPKG_TARGET_TRIPLET=x64-windows -DPROTOBUF_IMPORT_DIRS=$(Build.BinariesDirectory)\vcpkg\installed\x64-windows\include -DCMAKE_TOOLCHAIN_FILE=$(Build.BinariesDirectory)\vcpkg\scripts\buildsystems\vcpkg.cmake $(Build.SourcesDirectory)
+      cmake --build $(Build.BinariesDirectory)/Release --config MinSizeRel
+      cmake --install $(Build.BinariesDirectory)/Release --config MinSizeRel
+    displayName: 'Release Build'
+
+  - powershell: |
+      mkdir $(Build.BinariesDirectory)/Debug
+      cd $(Build.BinariesDirectory)/Debug
+      cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Debug -DVCPKG_ROOT=$(Build.BinariesDirectory)\vcpkg -DVCPKG_TARGET_TRIPLET=x64-windows -DPROTOBUF_IMPORT_DIRS=$(Build.BinariesDirectory)\vcpkg\installed\x64-windows\include -DCMAKE_TOOLCHAIN_FILE=$(Build.BinariesDirectory)\vcpkg\scripts\buildsystems\vcpkg.cmake $(Build.SourcesDirectory) 
+      cmake --build $(Build.BinariesDirectory)/Debug --config Debug
+      cmake --install $(Build.BinariesDirectory)/Debug --config Debug
+    displayName: 'Debug Build'
+
+  - script: |
+      7z a $(Build.SourcesDirectory)/RadialGM-MSVC-x64.exe -mmt -mx5 -sfx $(Build.SourcesDirectory)/Submodules/enigma-dev
+    displayName: 'Package'
+
+  - task: GithubRelease@0 
+    displayName: 'Create GitHub Release'      
+    inputs:
+      gitHubConnection: github.com_fundies
+      repositoryName: fundies/RadialGM
+      tagSource: manual
+      tag: RadialGM-$(Date:yyyyMMdd)$(Rev:.r)
```

Comment:
@RobertBColton any idea how I can make this match up with appveyor's tag? Cause my only idea is using date but I don't like that

Reply:
What's wrong with just $Rev? All we'd need is shared environment variable. If that's not doable Josh said before it's ok to check it into a text file.

Reply:
and how is appveyor going to know what azxure's $rev var is?

Reply:
I don't know what $rev is, can you tell me? It used to be used in LGM's about dialog too.
https://github.com/IsmAvatar/LateralGM/commit/76e25f9d519ceaf1d800eee74f06e417b5924d79
I removed it assuming it was an artifact of SVN and replaced it with a static string of LGM's version number so I could update the version number easier and uniformly.